client/watch.js

```diff
@@ -1036,6 +1069,22 @@ Template.context_browser.helpers({
   mediaTypeForDisplay (){
     return pluralizeMediaType(Session.get('mediaDataType') || Session.get('previousMediaDataType')).toUpperCase();
   },
+  replayAvailable(){
+    console.log(mainPlayer.activeStream.get());
+    
+    if(mainPlayer.activeStream.get().source === "youtube" && !mainPlayer.activeStream.get().live){
```

Comment:
Ah, I meant when an individual livestream becomes a not-livestream. This is probably a pain to test with youtube though. I recall testing it on Bambuser before. `replayAvailable` probably works assuming the "Live" indicator still works in that scenario.